src/index.ts

```diff
@@ -59,6 +59,7 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
   let standardHeaders: Record<string, string> = {}
   let fetchWithMiddleware = addMiddlewareToFetch(args => args)
   let eventSourceWithMiddleware = addMiddlewareToEventSource(args => args)
+  let lastCacheRefreshTime = null
```

Comment:
Best not to initialise as `null`, but rather leave as `undefined`, or you can simplify logic later by initialising as `0`
```suggestion
  let lastCacheRefreshTime: number
```